scripts/check_frontend_coverage.py

```diff
@@ -374,6 +374,7 @@
     'topic-editor-page.component.ts',
     'topic-editor-state.service.ts',
     'topic-editor-stories-list.directive.ts',
+    'topic-info-tab.directive.ts',
```

Comment:
Hi @marianazangrossi, @nithusha21, this line was added because I was not able to cover the following lines in `topic-info-tab.directive.ts`:
![image](https://user-images.githubusercontent.com/11008603/90723745-eae75b00-e2da-11ea-9517-500761a18547.png)


It seems possible to cover these lines by doing a `new TopicInfoTabDirective(dummyElementRef, dummyInjector)`. But throws an error when internal methods of the base class UpgradeComponent is called. Any suggestions on how to meaningfully cover these lines?

/cc @srijanreddy98

Reply:
You will have to create a component instance using TestBed to get this covered.

Reply:
Do I need to do `TestBed.configureTestingModule({..}).compileComponents()` prior to creating a component? If so, how do I specify a provider for `$rootScope` (which is injected in `topic-info-tab.directive.ts`)?

Reply:
Just to confirm this will be resolved before merge right?